### PR TITLE
Fix #36847 porno-chiks.com

### LINKED ISSUE #36847: porno-chiks.com
### Issue URL (Ads)
[NSFW][https://porno-chiks.com/videos/konchil-v-pizdu-miloy-devochke/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fporno-chiks.com%2Fvideos%2Fkonchil-v-pizdu-miloy-devochke%2F)
### Comment
Username: @spok1
> на втором снимке после становления на паузу

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>

### System configuration

Information | value
--- | ---
Platform: | Ext
AdGuard version: | 3.2.1
Browser: | Chrome
Stealth mode options: | Hide your search queries, Remove X-Client-Data header from HTTP requests, Strip URLs from tracking parameters, Hide your Referrer from third-parties
Filters: | AdGuard Russian,<br>AdGuard Annoyances,<br>AdGuard Base,<br>AdGuard Tracking Protection,<br>AdGuard Social Media

### PR DESCRIPTION
**Pull request by tvinzz**
https://github.com/AdguardTeam/AdguardFilters/issues/36847

### CODE REVIEW

File: RussianFilter/sections/specific.txt [hunk 1]
```diff
@@ -36,6 +36,8 @@ oka.fm##.rek-module
 pickpoint.ru##.bn-index-ad1
 mangalib.me##.ads
 gorlonos.com###DDWidgetSearch
+||porno-chiks.com/player/html.php?aid=pause*
```
Comment (slavaleleka) [inline RussianFilter/sections/specific.txt]: better add more specific rule
`||porno-chiks.com/player/html.php?aid=pause_html&video_id=*&*referer=`

**Review by slavaleleka — CHANGES_REQUESTED** [review]

File: RussianFilter/sections/specific.txt [hunk 2]
```diff
@@ -36,6 +36,8 @@ oka.fm##.rek-module
...
+||porno-chiks.com/contents/other/player/prerol.mp4
```
Comment (slavaleleka) [inline RussianFilter/sections/specific.txt]: and as a substitute rule for potential use in the future I propose 
`/player/prerol.mp4$domain=porno-chiks.com`

Comment (Alex-302) [reply RussianFilter/sections/specific.txt]: not sure if it popular)

File: RussianFilter/sections/specific.txt (hunk 1 above)
Comment (Alex-302) [reply RussianFilter/sections/specific.txt]: no, it is ok

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**